modules/caddyhttp/fileserver/caddyfile.go

```diff
@@ -59,6 +60,7 @@ func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error)
 //	    precompressed <formats...>
 //	    status        <status>
 //	    disable_canonical_uris
+//		file_limit	  <limit>
```

Comment:
It's strange, but gofmt tends to format code comments this way.
```suggestion
//	    file_limit      <limit>
```

Reply:
Yes, strange... I ran `go fmt` locally and it looks okay.

Reply:
This was a space / tab mismatch. I fixed it now.